src/picongpu/include/particles/ParticlesFunctors.hpp

```diff
@@ -166,6 +170,42 @@ struct CallUpdate
     }
 };
 
+/* Tests if species can be ionized and calls the function to do that */
+template<typename T_SpeciesName>
+struct CallIonization
+{
+    typedef T_SpeciesName SpeciesName;
+    typedef typename SpeciesName::type SpeciesType;
+    typedef typename SpeciesType::FrameType FrameType;
+
+    typedef typename HasFlag<FrameType, ionizer<> >::type hasIonizer;
+    
+    /* The following line resolves the alias of ionizer but does NOT 
+     * resolve its type, yet */
+    /* \todo I would like to do that via the particles/traits/GetIonizer */
```

Comment:
`GetIonizer` would feel sad if it wouldn't be used, now that it is implemented. I'm doing it.